src/Components/CheatSheet.js

```diff
@@ -1,10 +1,17 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
 import UserNavBar from "./UserNavBar";
+import SingleSign from "./SingleSign";
 
 class CheatSheet extends Component {
-  // pobranie wartosci z kana.json
+  state = {
+    kanaTable: "",
```

Comment:
I would start from `[]` here instead of `""` - if your user clicks "show" faster than the data loads this component will crash.  It will do so since `string` type does not have `.map` method and you call it in `render`.